cdap-app-fabric/src/test/java/io/cdap/cdap/internal/capability/CapabilityManagementServiceTest.java

```diff
@@ -169,7 +169,8 @@ public void testCapabilityManagement() throws Exception {
     //cleanup
     new File(externalConfigPath, fileName).delete();
     capabilityManagementService.runTask();
-    artifactRepository.deleteArtifact(Id.Artifact.from(new Id.Namespace(namespace), appName, version));
+    artifactRepository.clear(NamespaceId.SYSTEM);
```

Comment:
Can you move this to a `@AfterTest` method to make sure it gets cleanup after every test?